Source/WebGPU/WebGPU/CommandEncoder.h

```diff
@@ -117,6 +117,10 @@ class CommandEncoder : public WGPUCommandEncoderImpl, public RefCounted<CommandE
     void generateInvalidEncoderStateError();
     bool validateClearBuffer(const Buffer&, uint64_t offset, uint64_t size);
     static void trackEncoder(CommandEncoder&, WeakHashSet<CommandEncoder>&);
+#if ENABLE(WEBGPU_SWIFT) && defined(__WEBGPU__)
+    uint32_t maxTextureDimension1D();
```

Comment:
These could be const member functions I think

Reply:
I'll see if I can just make the m_device accesible to swift.